tests/12login/02cas.pl

```diff
@@ -0,0 +1,143 @@
+sub wait_for_cas_request
+{
+   my ( $expected_path, %params ) = @_;
+
+   await_http_request( $expected_path, sub {
+      return 1;
+   })->then( sub {
+      my ( $request ) = @_;
```

Comment:
Add a blank line after the line that unpacks `@_`; it helps visually separate it